modules/templates/helper.go

```diff
@@ -976,7 +976,7 @@ type remoteAddress struct {
 
 func mirrorRemoteAddress(ctx context.Context, m *repo_model.Repository, remoteName string) remoteAddress {
 	a := remoteAddress{}
-	if !m.IsMirror {
+	if m.IsMirror {
```

Comment:
Why is this changed?

Reply:
https://github.com/go-gitea/gitea/pull/18649#discussion_r896126862

Reply:
Aha, please open a new PR for that, so it can be included in the 1.17 version(this is currently targetting 1.18.0)

Reply:
here it is https://github.com/go-gitea/gitea/pull/20011

Reply:
That PR is merged, please remove the related change here.